pkg/monitoring/metrics/virt-controller/vmsnapshot.go

```diff
@@ -0,0 +1,42 @@
+package virt_controller
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+
+	io_prometheus_client "github.com/prometheus/client_model/go"
+	snapshotv1 "kubevirt.io/api/snapshot/v1beta1"
+)
+
+var (
+	vmSnapshotMetrics = []operatormetrics.Metric{
+		VmSnapshotSucceededTimestamp,
+	}
+
+	VmSnapshotSucceededTimestamp = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vmsnapshot_succeeded_timestamp_seconds",
+			Help: "Returns the timestamp of successful virtual machine snapshot.",
+		},
+		[]string{"vm_name", "snapshot_name", "namespace"},
+	)
+)
+
+func NewVMSnapshotCreated(snapshot *snapshotv1.VirtualMachineSnapshot) {
+	if snapshot.Status != nil &&
+		snapshot.Status.Phase == snapshotv1.Succeeded {
+		VmSnapshotSucceededTimestamp.WithLabelValues(
+			snapshot.Spec.Source.Name,
+			snapshot.Name,
+			snapshot.Namespace,
+		).Set(float64(snapshot.Status.CreationTime.Unix()))
+	}
+
+}
+
+func GetVMSnapshotMetric(vm, snapshot, namespace string) (float64, error) {
```

Comment:
```suggestion
func GetVmSnapshotSucceededTimestamp(vm, snapshot, namespace string) (float64, error) {
```

let's stay coherent with the metric name for this Getter